components/scripts/lib/build-scan-parse.sh

```diff
@@ -74,28 +73,48 @@ debug_build_scan_data() {
 
 # shellcheck disable=SC2034 # not all scripts use all of the fetched data
 parse_build_scan_row() {
-  local build_cache_metrics_only="$1"
-  local build_scan_row="$2"
+  local build_scan_row="$1"
 
   local run_num
 
   while IFS=, read -r run_num field_1 field_2 field_3 field_4 field_5 field_6 field_7 field_8 field_9 field_10 field_11 field_12 field_13 field_14 field_15 field_16 field_17 field_18 field_19 field_20 field_21; do
     debug "Build Scan $field_4 is for build $run_num"
-    project_names[run_num]="$field_1"
-    build_scan_ids[run_num]="$field_4"
-
-    if [[ "$build_cache_metrics_only" != 'build_cache_metrics_only' ]]; then
-      base_urls[run_num]="$field_2"
-      build_scan_urls[run_num]="$field_3"
-      git_repos[run_num]="$field_5"
-      git_branches[run_num]="$field_6"
-      git_commit_ids[run_num]="$field_7"
+
+    project_names[run_num]="${field_1}"
```

Comment:
Should we do the same (I mean the inverse) for project_names? (for example, to cover the case where the export API cannot be accessed, it would be nice to fall back to the project names derived from the folder names)

Reply:
I had made a change in #263 to always resolve the project name from the Gradle Enterprise API, and removed code relating to reading project names from Gradle and Maven. 

It seems that even after being reviewed and even after all the testing I did on the PR, the case where Gradle Enterprise API access is not available was not considered

I just tested it, and if API access is not configured, the project name will be unknown. 

![image](https://user-images.githubusercontent.com/5797900/231346784-4f602d14-3daa-4e88-93ee-4de9b78765db.png)

Do we want to add this functionality back? If so, then your comment makes sense to me.

Reply:
I suggest we do add it back as a fallback in case the EAPI cannot be called. But let's do that in a separate PR.